crypto_analytics/collection/data_source/crypto_compare.py

```diff
@@ -35,30 +38,41 @@ def fetch(self) -> pd.DataFrame:
         return self.data
 
     def write(self, filepath: str):
-        self.data.to_csv (filepath)
+        self.data.to_csv(filepath)
 
     def get_time(self):
         return self.data['time']
 
-    def _get_ochlv(self, ochlv_type: str, interval: Interval):
-        self.endpoint = self.valid_time_intervals.get(interval)
-        if not self.endpoint:
-            raise ValueError('Interval must be daily, hourly or minute')
+    def get_open(self):
+        return self.data['open']
 
-        ochlv = self.fetch()
-        return ochlv['Data'][ochlv_type]
+    def get_close(self):
+        return self.data['close']
 
-    def get_open(self, interval: Interval):
-        self._get_ochlv('open', interval)
+    def get_high(self):
+        return self.data['high']
 
-    def get_close(self, interval: Interval):
-        self._get_ochlv('close', interval)
+    def get_low(self):
+        return self.data['low']
 
-    def get_high(self, interval: Interval):
-        self._get_ochlv('high', interval)
+    # # TODO: determine whether to use volumeto or volumefrom
+    # def get_volume(self):
+    #     return self.data['volumeto']
```

Comment:
I didn't know what to do about this. crypto_compare doesn't have a volume column but has `volumeto` and `volumefrom`. That's why this class is inheriting from `OHLCDataSource` and not `OHLCVDataSource` until we know what to do here.

Reply:
```volumeto``` gives the volume of the crypto currency and ```volumefrom``` gives the volume of the corresponding fiat currency being used to buy that cryptocurrency (dollars, yen, euro, etc...).  Here we should probably use ```volumeto``` since this class is meant to gather the crypto currency data.  We could also add a method for each of them but that might make inheriting from ```OHLCVDataSource``` a little complicated.

Reply:
Oh ok that's much simpler than I thought. I thought it was like a range or something haha. Looking at the data though I think it's the other way around `volumefrom` seems to be the volume in crypto and `volumeto` the fiat. But of course that depends on what your `fsym` and `tsym` params are set to. But I think usually when we are looking at crypto charts we are converting 1 BTC -> $8xxx so `fsym = BTC` and `tsym = USD`. I'll add that change in a bit.

Reply:
Try running `python3 src/collection/pump_prediction_collect.py` and tell me if that looks right to you.

Reply:
Yup the output looks great!